virttest/utils_libvirt/libvirt_migration.py

```diff
@@ -0,0 +1,93 @@
+"""
+libvirt related migration functions
+"""
+import os
+import logging
+
+from virttest import defaults
+from virttest import libvirt_vm
+from virttest import remote
+from virttest.utils_test import libvirt
+
+
+def update_virsh_migrate_extra_opts(params):
```

Comment:
Remove this function because it is too simple and can be written in tp-libvirt case file.